source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -3,6 +3,121 @@ title: Migrate from our old frameworks
 weight: 210
 ---
 
-# Migrate from our old frameworks
+# Migrate to GOV.UK Frontend from our old frameworks
 
-Read about how to [update from GOV.UK Elements and Frontend Toolkit](https://test_moving_docs--govuk-design-system-preview.netlify.com/get-started/updating-your-code/).
+You should migrate to GOV.UK Frontend if you're using one of the following old ('legacy') frameworks:
+
+- GOV.UK Frontend Toolkit
+- GOV.UK Template
+- GOV.UK Elements
+
+We're no longer updating our old frameworks, and they do not meet the [Web Content Accessibility Guidelines (WCAG 2.1) accessibility standard](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).
```

Comment:
```suggestion
We're no longer updating our old frameworks, and they do not meet the [Web Content Accessibility Guidelines (WCAG 2.1 level AA) accessibility standard](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).
```

Reply:
Good spot, thanks! Will fix in the next commit.